modules/lvms-provisioning-storage-using-logical-volume-manager-operator.adoc

```diff
@@ -35,12 +41,16 @@ spec:
   resources:
     requests:
       storage: 10Gi <3>
-  storageClassName: lvms-vg1 <4>
+    limits:
+      storage: 20Gi <4>
+  storageClassName: lvms-vg1 <5>
 ----
 <1> Specify a name for the PVC.
 <2> To create a block PVC, set this field to `Block`. To create a file PVC, set this field to `Filesystem`.
-<3> Specify the storage size. The total storage size that you can provision is limited by the size of the Logical Volume Manager (LVM) thin pool and the overprovisioning factor.
-<4> The value of the `storageClassName` field must be in the format `lvms-<device_class_name>`, where, `<device_class_name>` is the value of the `deviceClasses.name` field in the `LVMCluster` CR.
+<3> Specify the storage size. If the value of this field is less than the minimum storage size, the requested storage size is
```

Comment:
```suggestion
<3> Specify the storage size. If the value is less than the set minimum storage size, the requested storage size is
```
I'd recommend removing "of this field" because it is already implied with the number callout

Reply:
Removed "of this field". 
In this case, the minimum storage size is not something that the users can set. It is there by default. It's just that users have to keep the min storage size values in mind while configuring this field.